app/presenters/two_factor_options_presenter.rb

```diff
@@ -63,4 +63,10 @@ def piv_cac_option
     return [] unless policy.available? || service_provider&.piv_cac_available?(current_user)
     [TwoFactorAuthentication::PivCacSelectionPresenter.new]
   end
+
+  def backup_code_option
+    policy = TwoFactorAuthentication::BackupCodePolicy.new(current_user)
+    return [TwoFactorAuthentication::BackupCodeSelectionPresenter.new] if policy.available?
+    []
```

Comment:
This needs coverage.

Reply:
I'm not sure why it's not getting it.  It should be now since this option is tested and available.

Reply:
Is this test triggering line 70?

Reply:
It's still showing no coverage for line 70